client/src/components/leaderboard/Leaderboard.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Please add this to .eslintrc rules
    "no-underscore-dangle": [2, { "allowAfterThis": true }],
it has been a pain in the ass, I use `_` for all private indicators.

Reply:
Looks like it is old comment for this file though.